context_test.go

```diff
@@ -686,6 +686,17 @@ func TestContextRenderNoContentSecureJSON(t *testing.T) {
 	assert.Equal(t, "application/json; charset=utf-8", w.HeaderMap.Get("Content-Type"))
 }
 
+func TestContextRenderNoContentAsciiJSON(t *testing.T) {
+	w := httptest.NewRecorder()
+	c, _ := CreateTestContext(w)
+
+	c.AsciiJSON(204, []string{"lang", "Go语言"})
```

Comment:
Please change `204` to `http` status code in HTTP package for maintainable.

Reply:
which status code should i use? you mean `http.StatusNoContent` or using other status code

Reply:
`http.StatusNoContent`

Reply:
done